reclass/adapters/ansible.py

```diff
@@ -75,15 +76,15 @@ def add_ansible_options_group(parser, defaults):
             apps = data['applications']
             if options.applications_postfix:
                 postfix = options.applications_postfix
-                groups.update([(k + postfix, v) for k,v in apps.iteritems()])
+                groups.update([(k + postfix, v) for k,v in apps.items()])
```

Comment:
So for this (and all like switches to items()) the two do not behave the same in python2 vs python3. I realize that this is the convenient fix but it produces code that behaves fundamentally differently between two versions (and is why six has an iteritems wrapper). You could probably fix this by importing future.utils and keeping iteritems()?